api/src/main/java/io/jsonwebtoken/lang/ImplementationNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package io.jsonwebtoken.lang;
+
+import io.jsonwebtoken.JwtException;
+
+/**
+ * Exception indicating that no implementation of an jjwt-api SPI was found on the classpath.
+ */
+public class ImplementationNotFoundException extends JwtException {
```

Comment:
I don't know why, but I've never liked the JDK classes that are like FooNotFoundException or WhateverNotBlahException - I've always felt like they didn't flow well and were clunky for the English language.

Can we make this `UnavailableImplementationException`?

Also, should this extend from some other exception instead of `JwtException` directly?